radio/src/opentx.cpp

```diff
@@ -538,6 +538,11 @@ void checkBacklight()
         resetBacklightTimeout();
       }
     }
+    #if defined(HARDWARE_TOUCH)
```

Comment:
With the other touch screens, this is done in the ISR, so that it will be reset twice on these radios. Can you please add a NV14 ifdef? (needs to be unified, this is probably the right place anyway).

Reply:
Done